onadata/apps/api/viewsets/user_profile_viewset.py

```diff
@@ -95,6 +98,46 @@ def set_is_email_verified(profile, is_email_verified):
     profile.save()
 
 
+def check_user_lockout(request):
+    username = request.user.username
+    lockout = cache.get('{}{}'.format(LOCKOUT_CHANGE_PASSWORD_USER, username))
+    if lockout:
+        time_locked_out = \
+            datetime.now() - datetime.strptime(lockout,
+                                               '%Y-%m-%dT%H:%M:%S')
+        remaining_time = round(
+            (getattr(settings, 'LOCKOUT_TIME', 1800) -
+             time_locked_out.seconds) / 60)
+        response = {'error' :
+            'Too many password reset attempts, Try again in {} minutes'.format(
+                remaining_time)}
+        return response
+
+
+def change_password_attempts(request):
+    """Track number of login attempts made by user within a specified amount
+     of time"""
+    username = request.user.username
+    attempts = cache.get('{}{}'.format(CHANGE_PASSWORD_ATTEMPTS, username))
```

Comment:
The expression `'{}{}'.format(CHANGE_PASSWORD_ATTEMPTS, username)` is repeated several times in this function, can we have it assigned to a variable and then used instead? It will be simpler to make modifications in one place than to look for all the places it appears. 

```python
your_variable = '{}{}'.format(CHANGE_PASSWORD_ATTEMPTS, username)
...
cache.incr(your_variable)
```